plumgrid_playbooks/plumgrid_cleanup.yml

```diff
@@ -19,23 +19,41 @@
 #          PLUMgrid environment
 #
 
+# Remove Libnetwork Plugin and Docker
+- name: Cleanup PG CLI, Libnetwork Plugin & Docker
+  hosts: edge_hosts
+  user: root
+  tasks:
+    - name: Remove PG CLI, Libnetwork Plugin and Docker
+      ignore_errors: yes
+      apt:
+        name: "{{ item }}"
+        state: absent
+        purge: yes
+      with_items:
+        - pg-cli
+        - libnetwork
+        - docker-engine
+
 # Remove PLUMgrid Edges & Directors
 - name: Cleanup PLUMgrid
   hosts: director_hosts:edge_hosts:gateway_hosts
   user: root
   tasks:
-    - name: remove sources.list
-      file: path=/etc/apt/sources.list.d/plumgrid.list state=absent
-    - name: unload iovisor
-      command: /sbin/rmmod iovisor
-    - name: remove iovisor dkms
-      apt:
-        name: iovisor-dkms
+    - name: Remove sources.list
+      file:
+        path: "/etc/apt/sources.list.d/plumgrid.list"
         state: absent
-        purge: yes
-    - name: remove plumgrid-lxc
+
+    - name: Unload iovisor
+      command: /sbin/rmmod iovisor
+
+    - name: Remove iovisor-dkms and plumgrid-lxc
       apt:
-        name: plumgrid-lxc
+        name: "{{ item }}"
         state: absent
         purge: yes
         force: yes
+      with_items:
+        - iovisor-dkms
+        - plumgrid-lxc
```

Comment:
this needs to be updated to plumgrid-edge-docker for edges and plumgrid-lxc for directors. make a variable that defines pg-package name for these nodes so it can updated according to packaging